section1/ex5.rb

```diff
@@ -0,0 +1,22 @@
+name = 'Zed A. Shaw'
+age = 35 # not a lie in 2009
+height = 74 # inches
+weight = 180 # lbs
+eyes = 'Blue'
+teeth = 'White'
+hair = 'Brown'
+metric_height= height * 2.54
+metric_weight= weight * 0.453592
+
+puts "Let's talk about #{name}."
+puts "He's #{height} inches tall."
+puts "He's #{weight} pounds heavy."
+puts "Actually that's not too heavy."
+puts "He's got #{eyes} eyes and #{hair} hair."
+puts "His teeth are usually #{teeth} depending on the coffee."
+
+# this line is tricky, try to get it exactly right
+puts "If I add #{age}, #{height}, and #{weight} I get #{age + height + weight}."
+
+puts "His height in cm's is #{metric_height}"
+puts "His weight in kg's is #{metric_weight}"
```

Comment:
Good work

Challenge yourself: if you run this, the last line prints 5 decimal places. How could you update the code to limit the decimals to only 2?